charters/rust-core/README.md

```diff
@@ -0,0 +1,277 @@
+- Codification of the API Design using the Custom DSL.
+- Example usages.
+- Doc comments inline for the API Design.
+- Test vector resources.
+
+**Timeline/Milestones**: Unknown???
+
+**Exit Criteria**: Comprehensive feature coverage for the tbDEX specification, codified in the Custom DSL, committed to source code.
+
+#### [Deliverable 7]: tbDEX Rust Implementation
+
+**Description**: Rust implementation of the [[Deliverable 6]: tbDEX API Design (APID)](#deliverable-6-tbdex-api-design-apid), intended for public consumption.
+
+**DRI**: Diane Huxley (@diehuxx).
+
+**Scope of Work**:
+
+- Full conformance to [[Deliverable 6]: tbDEX API Design (APID)](#deliverable-6-tbdex-api-design-apid).
+- Comprehensive integration of test vectors (from APID).
+- Comprehensive documentation comments (from APID).
+- CI/CD implementations for security analysis, test automation, and crate publication.
+
+**Timeline/Milestones**: ???
+
+**Exit Criteria**: Full implementation of *Scope of Work* as a Rust project.
+
+#### [Deliverable 8]: tbDEX UniFFI Binding
+
+**Description**: Rust crate which utilizes [UniFFI](https://github.com/mozilla/uniffi-rs) in conjunction with the [[Deliverable 7]: tbDEX Rust Implementation](#deliverable-7-tbdex-rust-implementation) to create Kotlin and Swift binded code, which feed into [[Deliverable 9]: tbDEX Kotlin Implementation](#deliverable-9-tbdex-kotlin-implementation) and [[Deliverable 10]: tbDEX Swift Implementation](#deliverable-10-tbdex-swift-implementation). This deliverable is intended primarily for internal purposes but does not exclude the possibility of public consumption.
+
+**DRI**: Kendall Weihe (@KendallWeihe).
+
+**Scope of Work**:
+
+- UDL file with comprehensive feature coverage from [[Deliverable 7]: tbDEX Rust Implementation](#deliverable-7-tbdex-rust-implementation).
+- Successful binding executions for Kotlin & Swift.
+
+**Timeline/Milestones**: ???
+
+**Exit Criteria**: Successful binding executions, over the comprehensive set of features from [[Deliverable 7]: tbDEX Rust Implementation](#deliverable-7-tbdex-rust-implementation), for Kotlin and Swift.
+
+#### [Deliverable 9]: tbDEX Kotlin Implementation
+
+**Description**: Kotlin implementation of tbDEX by wrapping [[Deliverable 8]: tbDEX UniFFI Binding](#deliverable-8-tbdex-uniffi-binding) as to be conformant to the [[Deliverable 6]: tbDEX API Design (APID)](#deliverable-6-tbdex-api-design-apid), intended for public consumption.
+
+**DRI**: Neal Roessler (@nitro-neal).
+
+**Scope of Work**:
+
+- Utilization of the output of the [[Deliverable 8]: tbDEX UniFFI Binding](#deliverable-8-tbdex-uniffi-binding).
+- Full conformance to [[Deliverable 6]: tbDEX API Design (APID)](#deliverable-6-tbdex-api-design-apid).
+- Comprehensive integration of test vectors (from APID).
+- Comprehensive documentation comments (from APID).
+- CI/CD implementations for security analysis, test automation, and crate publication.
+
+**Timeline/Milestones**: ???
```

Comment:
this date must be tied to our internal deadlines, fyi

Reply:
Yes and no. What's being proposed with the WG has a higher set of expectations than early (& narrower) use cases. AKA we can cut corners with the early use cases on things like documentation, test vectors, CI/CD, etc. Early use cases have a high-bar with respect to expectations, I don't mean to imply the inverse, because we have to establish high levels of confidence, but early use cases have a smaller footprint than the scope of work for the WG (& the deliverable specifically commented on here).